Sources/PackageGraph/PinsStore.swift

```diff
@@ -6,15 +6,15 @@
 
  See http://swift.org/LICENSE.txt for license information
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ */
 
+import PackageModel
+import SourceControl
 import TSCBasic
 import TSCUtility
-import SourceControl
-import PackageModel
 
 public final class PinsStore {
-    public typealias PinsMap = [PackageReference.PackageIdentity: PinsStore.Pin]
+    public typealias PinsMap = [String: PinsStore.Pin]
```

Comment:
on the surface, feels like the key would be better if it stayed based on identity. what led to changing it to a string?

Reply:
`PackageReference.PackageIdentity` is/was a typealias to `String`, so this diff is functionally equivalent. I agree, though, that `PinsStore` should be keyed on `PackageIdentity`, and look forward to doing that alongside the broader semantic change.

Reply:
sorry so should this be PackageIdentity instead of String?